lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -1487,6 +1487,18 @@ def _resource_sku_to_capability(  # noqa: C901
         if not node_space.network_interface.max_nic_count:
             node_space.network_interface.max_nic_count = 8
 
+        # For Dp/Ep_v5 VM size, the `Accelerated Networking` is required. But the API
```

Comment:
Dp/Ep_v5 VM sizes have `Accelerated Networking` capability.

One core size also supports?

Reply:
@lubaihua33 please double check this. I don't have other concerns.

Reply:
> Dp/Ep_v5 VM sizes have `Accelerated Networking` capability.
> 
> One core size also supports?

For Dp/Ep_V5 VM sizes, the smallest size has 2 cores. I tried one core in Hyper-V. One core also supports SRIOV.